src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    private static final String COLUMN_ID = "id";
+    private static final String COLUMN_NAME = "name";
+    private static final String LICENCE = "licence_number";
+    private static final String STATEMENT_UPDATE = "UPDATE drivers "
+            + "SET " + COLUMN_NAME + " = ?, " + LICENCE + " = ? "
+            + "WHERE id = ? " + "AND is_deleted IS FALSE;";
+    private static final String STATEMENT_CREATE = "INSERT "
+            + "INTO drivers(" + COLUMN_NAME + "," + LICENCE + ") "
+            + "Values(?,?);";
+    private static final String STATEMENT_GET = "SELECT * "
+            + "FROM drivers "
+            + "WHERE id = ? "
+            + "AND is_deleted IS FALSE;";
+    private static final String STATEMENT_GETALL = "SELECT * "
+            + "FROM drivers "
+            + "WHERE is_deleted IS FALSE;";
+    private static final String STATEMENT_DELETE = "UPDATE drivers "
+            + "SET is_deleted = true "
+            + "WHERE id = ? "
+            + "AND is_deleted IS FALSE;";
+    private static final String GET_FAILED = "Failed to obtain driver with id: ";
+    private static final String CREATE_FAILED = "Failed to create new driver with name= ";
+    private static final String UPDATE_FAILED = "Failed to update driver with id: ";
+    private static final String DELETE_FAILED = "Failed to delete driver with id: ";
+    private static final String GET_ALL_FAILED = "Failed to read all drivers from db.";
+
+    @Override
+    public Driver create(Driver driver) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createStatement = connection.prepareStatement(
+                        STATEMENT_CREATE, Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, driver.getName());
+            createStatement.setString(2, driver.getLicenceNumber());
+            createStatement.executeUpdate();
+            ResultSet resultSet = createStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+                return driver;
+            } else {
+                throw new DataProcessingException(GET_FAILED);
```

Comment:
the exception message will be uninformative. Maybe it's better to also add an id?